pkg/specgen/namespaces.go

```diff
@@ -381,53 +369,46 @@ func ParseNetworkFlag(networks []string, pastaNetworkNameExists bool) (Namespace
 	case ns == string(Host):
 		toReturn.NSMode = Host
 	case strings.HasPrefix(ns, "ns:"):
-		split := strings.SplitN(ns, ":", 2)
-		if len(split) != 2 {
-			return toReturn, nil, nil, errors.New("must provide a path to a namespace when specifying \"ns:\"")
-		}
+		_, value, _ := strings.Cut(ns, ":")
```

Comment:
Changed this code to preserve existing behavior, the split always works because of `HasPrefix`, but maybe error message was intended for `len(value) == 0`